examples/motor/m2006_double_continue_shoot.cc

```diff
@@ -0,0 +1,120 @@
+/****************************************************************************
+ *                                                                          *
+ *  Copyright (C) 2022 RoboMaster.                                          *
+ *  Illini RoboMaster @ University of Illinois at Urbana-Champaign          *
+ *                                                                          *
+ *  This program is free software: you can redistribute it and/or modify    *
+ *  it under the terms of the GNU General Public License as published by    *
+ *  the Free Software Foundation, either version 3 of the License, or       *
+ *  (at your option) any later version.                                     *
+ *                                                                          *
+ *  This program is distributed in the hope that it will be useful,         *
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of          *
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the           *
+ *  GNU General Public License for more details.                            *
+ *                                                                          *
+ *  You should have received a copy of the GNU General Public License       *
+ *  along with this program. If not, see <http://www.gnu.org/licenses/>.    *
+ *                                                                          *
+ ****************************************************************************/
+
+// #define WITH_CONTROLLER
+
+#include "bsp_gpio.h"
+#include "bsp_os.h"
+#include "bsp_print.h"
+#include "cmsis_os.h"
+#include "controller.h"
+#include "main.h"
+#include "motor.h"
+#include "utils.h"
+#include "dbus.h"
+
```

Comment:
Need to add descriptive comments on how to use this example. (What switch control what; what is the expected behavior, etc.)